server/routes/my_garden.js

```diff
@@ -1,23 +1,33 @@
 const express = require("express");
 const router = express.Router();
-const { getUserPlantInfo, deleteUserPlant } = require("../../db/query_functions.js");
+const { getUserPlantInfo, deleteUserPlant, addPlantToMyGarden } = require("../../db/query_functions.js");
 
 module.exports = (db) => {
   router.get('/', (req, res) => {
-    res.send("This would be the user_plants page");
+    res.send("This would be the my_garden page");
+  });
+
+  // Add plant to user's garden (from plant library)
+  router.post("/submit", (req, res) => {
+    let newPlantId = req.body.plantId;
+    const userId = 1;
+    
+    const [queryString, values] = addPlantToMyGarden(newPlantId, userId);
+
+    db.query(queryString, values).then((data) => {
+      res.send();
+    });
   });
 
   router.get("/:id", (req, res) => {
     db.query(getUserPlantInfo(req.params.id)).then((data) => {
-      console.log(data);
       res.json(data.rows);
     });
   });
 
   router.delete("/:id", (req, res) => {
     db.query(deleteUserPlant(req.params.id)).then(() => {
       res.send();
-      // Do I need to send anything here?
```

Comment:
The server should send a status code of success (I forget the number) after a successful delete call

Reply:
Status 200 added